README.md

```diff
@@ -56,37 +57,30 @@ Browse the [HASH roadmap] for more information about upcoming features and relea
 
 This repository's contents is divided across four primary sections:
 
-- [`/apps` - applications](#applications): the primary code behind our runnable applications
-- [`/blocks` - blocks](#blocks): our public [Block Protocol] blocks
-- [`/infra`- infrastructure](#infrastructure): deployment scripts and other tools used to run our apps
-- [`/libs` - libraries](#libraries): includes npm packages and Rust crates
+- [**`/apps`**](/apps) contains the primary code powering our runnable [applications](#applications)
+- [**`/blocks`**](/blocks) contains our public _Block Protocol_ [blocks](#blocks)
+- [**`/infra`**](/infra) houses deployment scripts, utilities and other [infrastructure](#infrastructure) useful in running our apps
+- [**`/libs`**](/libs) contains all of our public [libraries](#libraries), including npm packages and Rust crates
 
-### Applications
-
-#### HASH
-
-- [`/apps/hash`](apps/hash): entrypoint for [HASH] - a data-driven, entity-centric, all-in-one workspace based on the Block Protocol
+Key projects within are summarized below.
 
-#### HASH Simulations
-
-- [`/apps/engine`](apps/engine): codebase for our next-gen version of [HASH Engine] - a versatile agent-based simulation engine written in Rust
-
-#### Our websites
+### Applications
 
-- [`/apps/hashdotai`](apps/hashdotai): contains the [HASH user guide] and [glossary of terms] content
```

Comment:
This removes the links to `hash.ai/docs` (line 7) and `hash.ai/glossary` (line 8), so `markdownlint` fails.